src/sessions.py

```diff
@@ -24,7 +24,7 @@ def __init__(self, app):
         # Extra security
         self.app.config['SESSION_COOKIE_HTTPONLY'] = True
         self.app.config['SESSION_COOKIE_SECURE'] = True
-        self.app.config['SESSION_COOKIE_SAMESITE'] = 'Lax' # Option: 'Strict'
+        self.app.config["SESSION_COOKIE_SAMESITE"] = "None"  # Option: 'Lax', 'Strict'
```

Comment:
I believe this can be reverted. We saw a warning that mentioned that the app would function as if 'Lax' were in the config here because it didn't understand "None". This could be tested.

Reply:
We tried reverting this change(using 'Lax'). We got an error(or maybe warning?) that stated that some function couldn't be done because this was set to 'Lax'.